make/ToolsJdk.gmk

```diff
@@ -82,6 +82,11 @@ TOOL_GENERATEEXTRAPROPERTIES = $(JAVA_SMALL) -cp $(BUILDTOOLS_OUTPUTDIR)/jdk_too
 TOOL_MAKEZIPREPRODUCIBLE = $(JAVA_SMALL) -cp $(BUILDTOOLS_OUTPUTDIR)/jdk_tools_classes \
     build.tools.makezipreproducible.MakeZipReproducible
 
+TOOL_JIMAGE_DIFF_TO_JMODS = $(BUILD_JAVA_SMALL) -cp $(BUILDTOOLS_OUTPUTDIR)/jdk_tools_classes \
+    --add-modules=jdk.jlink --add-exports=java.base/jdk.internal.module=ALL-UNNAMED \
+    --add-exports=java.base/jdk.internal.jimage=ALL-UNNAMED \
+    build.tools.runtimelink.JimageDiffGenerator
```

Comment:
While it might be a bit redundant, we try to keep the same name in the make name, the package and the main class, e.g. something like:

```suggestion
TOOL_JIMAGE_DIFF_GENERATOR = $(BUILD_JAVA_SMALL) -cp $(BUILDTOOLS_OUTPUTDIR)/jdk_tools_classes \
    --add-modules=jdk.jlink --add-exports=java.base/jdk.internal.module=ALL-UNNAMED \
    --add-exports=java.base/jdk.internal.jimage=ALL-UNNAMED \
    build.tools.jimagediffgenerator.JimageDiffGenerator
```

This is of course not consistently followed, but for new tooling I think it would be a good idea to try and follow.

Reply:
Based on some private feedback I've got it seems this will change a bit (move to the `jdk.jlink` module instead). But if still relevant, I'll keep that in mind. Thanks!

Reply:
The latest update does the compilation in `make/Images.gmk` and doesn't use the tool approach any more. This should be moot.